Exception/PriceFormatterException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
```

Comment:
yes, should be part of every file. i'm going to change that in every file that i touch.